src/routes/safe/store/actions/transactions/utils/transferDetails.ts

```diff
@@ -0,0 +1,84 @@
+import { Transfer, TxConstants } from 'src/routes/safe/store/models/types/transactions'
+import { sameAddress } from 'src/logic/wallets/ethAddresses'
+import { store } from 'src/store'
+import { safeParamAddressFromStateSelector } from 'src/routes/safe/store/selectors'
+import {
+  ERC20TransferDetails,
+  ERC721TransferDetails,
+  ETHTransferDetails,
+  UnknownTransferDetails,
+} from './transferDetails.d'
+import { humanReadableValue } from 'src/logic/tokens/utils/humanReadableValue'
+
+const isIncomingTransfer = (transfer: Transfer): boolean => {
+  const state = store.getState()
+  const safeAddress = safeParamAddressFromStateSelector(state)
+  return sameAddress(transfer.to, safeAddress)
+}
```

Comment:
not really a fan of accessing store like this, this should be done only if there's no other option. why couldn't we just pass safeAddress there?

Reply:
actually this is not being used, as `transfers` only come as part of the `allTransactions` returned data.

I added a TODO here, so we can address this in the future